src/main/scala/it/pagopa/interop/catalogmanagement/model/persistence/CatalogPersistentBehavior.scala

```diff
@@ -105,6 +105,20 @@ object CatalogPersistentBehavior {
             Effect.none[CatalogItemDocumentDeleted, State]
           }
 
+      case AddCatalogItemRiskAnalysis(eServiceId, catalogRiskAnalysis, replyTo) =>
+        val catalogItem: Option[CatalogItem] = state.items.get(eServiceId)
+
+        catalogItem
+          .map { _ =>
+            Effect
+              .persist(CatalogItemRiskAnalysisAdded(eServiceId, catalogRiskAnalysis))
+              .thenRun((_: State) => replyTo ! success(Some(catalogRiskAnalysis)))
+          }
+          .getOrElse {
+            replyTo ! success(None)
```

Comment:
any reason why this is not
```suggestion
            replyTo ! StatusReply.error[Done](EServiceNotFound(eServiceId))
```
?